docs/walkthrough.md

```diff
@@ -32,17 +32,21 @@ be done with Helm setup.
 If you don't already have Helm v2, see the
 [installation instructions](https://github.com/kubernetes/helm/blob/master/docs/install.md).
 
-### RBAC Considerations
-
-If your kubernetes cluster has [RBAC](https://kubernetes.io/docs/admin/authorization/rbac/)
-enabled, you must ensure that the default service account for the `kube-system`
-namespace has the `cluster-admin` role:
+If your kubernetes cluster has
+[RBAC](https://kubernetes.io/docs/admin/authorization/rbac/) enabled, you must
+ensure that tiller pod has `cluster-admin` access. The default install of tiller
+done with `helm init` installs the tiller pod into `kube-system` namespace and
```

Comment:
> "install .... done ... with `helm init` installs the tiller pod into ... and uses ..."

doesn't sound grammatically correct to me, and it's also hard to get the meaning.
I would rephrase it to something like:
> By default, the `helm init` command installs the tiller pod into `kube-system` namespace, and uses the `default` service account.